cove/settings.py

```diff
@@ -86,6 +86,16 @@
                       'record': 'record-package-schema.json'},
         'default': None
     },
+    'schema_version': {  # Default schema version
+        'cove-ocds': '1.0.2',
+        'cove-360': None,
+        'default': None
+    },
+    'schema_version_choices': {
+        'cove-ocds': ('1.0.2', '1.0.1', '1.0.0'),  # Keep descending order when adding new versions
```

Comment:
I think having patch versions here is temporary, so that we can test this. Is that right?

I think for the end of this sprint we want 1.0 and the 1.1 dev version.
I'm not sure how best to implement it, but one way would be to have:
`SCHEMA_URL_OCDS=http://standard.open-contracting.org/{}/en/`
and the list of tuples as `('1.1-dev', '1.0')`.

We probably need to discuss this with @kindly